go/enclave/components/batch_registry.go

```diff
@@ -279,6 +280,14 @@ func (br *batchRegistryImpl) BatchesAfter(batchHash gethcommon.Hash) ([]*core.Ba
 	}
 
 	for batch.Number().Cmp(headBatch.Number()) != 0 {
+		// todo @stefan - move this somewhere else, it shouldn't be in the batch registry.
+		rollupLimiter := limiters.NewRollupLimiter(limiters.MaxTransactionSize)
```

Comment:
shouldn't this limiter be created outside the loop? It's only checking the one batch each time instead of the accumulation.

Reply:
Good spot